android/README.md

```diff
@@ -0,0 +1,443 @@
+# Asgardeo Android SDK
+
+## Introduction
+
+The Asgardeo Auth Android SDK enables Android applications (written in Kotlin) to utilize OpenID Connect (OIDC) authentication with the Asgardeo serving as the Consumer Identity and Access Management (CIAM) Provider through application-native authentication. This SDK assists you in creating custom login flows directly within the applications themselves, without relying on browser redirects, thereby prioritizing user experience.
+
+## Prerequisite
+
+- Supports Android applications written in Kotlin programming language.
+- The minimum supported SDK is API level 26, compiled to API level 34. However, there are certain limitations when using specific authentication methods:
+    - Passkeys are only supported on API level 34 and above.
+    - Google authentication using the Credential Manager API is supported on API level 34 and above.
+
+## Getting Started
+
+### Register your application in Asgardeo
+
+#### Asgardeo
+
+1. [Register to Asgardeo and create an organization if you don't already have one](https://wso2.com/asgardeo/docs/get-started/create-asgardeo-account/). The organization name you choose will be referred to as `<org_name>` throughout this document.
+2. [Register a Mobile Application in Asgardeo to integrate your application with Asgardeo](https://wso2.com/asgardeo/docs/guides/applications/register-mobile-app/). You will obtain a `client_id` from Asgardeo for your application which will need to be embedded later for the SDK integration. Also note the redirect URI that you used to create the application, this is also required for the SDK integration.
+3. In the created mobile application, go to the "Advanced" section and [enable the application native authentication for your Android application](https://is.docs.wso2.com/en/latest/guides/authentication/add-application-native-login/#enable-app-native-authentication).
+
+### Installing the SDK
+
+Add the latest released SDK in the `build.gradle` file of your Android application.
+
+```groovy
+dependencies {
+    implementation 'io.asgardeo:android:1.0.0'
+}
+```
+
+Add a redirect scheme in the Android application. You need to add the `appAuthRedirectScheme` in the application `build.gradle` file.
+
+This should be consistent with the CallBack Url of the Service Provider that you configured in the Asgardeo.
+
+For example, if you have configured the `callBackUrl` as `wso2sample://oauth2`, then the `appAuthRedirectScheme` should be `wso2sample`.
+
+```groovy
+android.defaultConfig.manifestPlaceholders = [
+    'appAuthRedirectScheme': 'wso2sample'
+]
+```
```

Comment:
Shall we make this a numbered list to clearly outline the steps?

Reply:
Fixed with https://github.com/asgardeo/mobile-ui-sdks/pull/1/commits/fe2efb415d877fcf11b14155642113e751069e6e